### PR TITLE
[REF] Replaces master with main where possible

### LINKED ISSUE #619: Set default branch to main
### Summary
We have changed the default branch from `master` to `main` per unanimous agreement among the steering committee. Please contact me at joshua.teves@nih.gov if you have any issues with the switch. GitHub's recommended method of switching your local repository is the following:
```
git branch -m master main
git fetch origin
git branch -u origin/main main
```
HOWEVER, it may be the case that your personal fork is `origin` and the main repository is `upstream` (or some similar variant). In that case, please do the following:
1. Switch your branch from `master` to `main` in your repository Settings under "Branches." (Note: GitHub will already alert you that the main repository has changed and encourage you to do so.)
2. In your local repository, run the following:
```
git branch -m master main
git fetch upstream main
git push origin main
```

### Next Steps
- [x] Solicit feedback
- [x] Identify places where the codebase and repository would have to modified to accommodate this change
- [x] Make changes as needed

### PR DESCRIPTION
**Pull request by jbteves**
Addresses but does not close #619 .

Changes proposed in this pull request:

- Switch `master` to `main` where applicable
-

NOTE: due to versioneer and the document doc tree, there are some references to `master` that cannot be changed from my read. All examples below:
```
docs/conf.py:67:# The master toctree document.
docs/conf.py:68:master_doc = 'index'
docs/governance.rst:184:`decision-making rules for the BIDS standard <https://github.com/bids-standard/bids-specification/blob/master/DECISION-MAKING.md>`_,
tedana/_version.py:192:        # "stabilization", as well as "HEAD" and "master".
versioneer.py:18:(https://travis-ci.org/warner/python-versioneer.png?branch=master)
versioneer.py:179:  "master" and "slave" subprojects, each with their own `setup.py`,
versioneer.py:612:        # "stabilization", as well as "HEAD" and "master".
versioneer.py:1004:        # "stabilization", as well as "HEAD" and "master".
```

### CODE REVIEW

**Review by tsalo — APPROVED** [review]
LGTM. Thanks for doing this.

File: CONTRIBUTING.md
Comment (emdupre) [inline CONTRIBUTING.md:326]: This doesn't resolve for me. I think we're stuck with the old one:
```suggestion
[restructuredtext]: http://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html
```

File: CONTRIBUTING.md
Comment (emdupre) [inline CONTRIBUTING.md:327]: This also doesn't resolve :\ 
```suggestion
[sphinx]: http://www.sphinx-doc.org/en/master/index.html
```

File: tedana/due.py
Comment (emdupre) [inline tedana/due.py:13]: Will need to revert to resolve:
```suggestion
See  https://github.com/duecredit/duecredit/blob/master/README.md for examples.
```

**Review by emdupre — COMMENTED** [review]
Thanks for this, @jbteves ! 

I'm getting three external links that don't (yet) resolve with `main`, so I think we'll need to revert those changes for now. But overall, very happy to have this updated !!

**Comment by jbteves** [comment]
Oh, silly me, good catches @emdupre !

**Comment by jbteves** [comment]
I think the issue was that at some point GitHub had stated they would redirect `main` and `master` correctly for you, but I didn't check and it seems that's not implemented. My fault for not checking.

**Comment by emdupre** [comment]
> I think the issue was that at some point GitHub had stated they would redirect main and master correctly for you, but I didn't check and it seems that's not implemented. My fault for not checking.

No worries ! I was confused on that too -- the scikit link redirects correctly from `master` to `main`, but I think that's only because they've made a `main` branch already. The remaining links still only have the `master` branch, it seems. Regardless, I think these changes are now ready to go !

**Review by emdupre — APPROVED** [review]

**Comment by jbteves** [comment]
So my plan to proceed, once we have approvals, is to do the following:
- Merge this PR.
- Throw the switch from `master` to `main`.
- Pin #667 , with the top comment edited with the instructions and reiterating that I'm the point of contact if developers need help.
- If issues arise, try to tackle them tonight.

If there's no objections, then pending the additional review from @tsalo I'll go ahead.

**Review by tsalo — APPROVED** [review]

**Review by eurunuela — APPROVED** [review]
LGTM! Thank you @jbteves . If we miss something we'll see after merging anyway. I'll be checking my notifications in case you need any help.

**Comment by tsalo** [comment]
I've done the rename, but I'll also make sure you have permissions for the future.

**Pull request merged**
